pkg/operator/controller/nodeobservability/controller.go

```diff
@@ -202,7 +206,7 @@ func (r *NodeObservabilityReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 	// if machine config change is not requested, we can mark it as ready
 	var nomcReady bool = true
```

Comment:
```suggestion
	var mcReady bool = true
```

Would be good to modify the log message on the line 218 from `NodeObservabilityMachineConfig` to something like `MachineConfig` or `MachineConfiguration`. As now the machine can be ready without `NOBMC` CR being involved.